### DISCUSSION TITLE
2024-08-21 회의

### DISCUSSION THREAD

**Discussion by chanhihi** [jiphyeonjeon-42/frontend#615] [meeting]
@mixsung
@hyeunkim42 
@chanhihi 

개발 환경설정.
개발 방향성.

**Comment by mixsung** [comment]
개발 방향성
- 최적화 및 성능 개선 경험을 하고 싶다!
- 리액트를 사용하면서 리액트의 작동방식, 원리 등 자세하게 파보고 싶다. (프론트 팀의 자체 스터디 진행 등)

**Reply by chanhihi** [reply]
<img width="1919" alt="image" src="https://github.com/user-attachments/assets/6f067969-b7f2-492f-92f1-0d798fda72a2">

**Reply by chanhihi** [reply]
1. 라이트하우스를 고려하여 성능개선 리팩토링을 진행하시면 좋을 것 같습니다.

**Comment by hyeunkim42** [comment]
개발 방향성
- 웹 개발 찍먹해보기
- 사용자 편리성을 고려한 기능 추가

**Reply by hyeunkim42** [reply]
기능 예시
카테고리 별 검색
청구기호로 검색 (사서측)

**Comment by chanhihi** [comment]
1. 리액트 구조개선 프론트엔드단에서의 클린 아키텍쳐.
2. 모노레포를 만약에 도입하신다면, 도입 이점이나 오버엔지니어링이면 왜 그런가를 좀 생각해서 고려를 해보시는 것이 좋을 거 같습니다.
3. CI / CD 백엔드에서 뺏어 오십쇼.
4. 테스팅코드를 추가 한다면 좋을거 같습니다. (cf. monorepo)
